src/Uintra20/Content/uintra20/src/app/ui/pages/uintra-groups/create/uintra-groups-create-page.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { finalize } from 'rxjs/operators';
+
+@Component({
+  selector: 'uintra-groups-create-page',
+  templateUrl: './uintra-groups-create-page.html',
+  styleUrls: ['./uintra-groups-create-page.less'],
+  encapsulation: ViewEncapsulation.None
+})
+export class UintraGroupsCreatePage {
+  data: any;
+  title: string = "";
+  description: string = "";
+  files: any[] = [];
+  isShowValidation: boolean = false;
+  inProgress: boolean = false;
+
+  constructor(
+    private route: ActivatedRoute,
+    private http: HttpClient
+  ) {
+    this.route.data.subscribe(data => this.data = data);
+  }
+
+  get isSubmitDisabled() {
+    return this.inProgress;
+  }
+
+  ngOnInit() {
+
+  }
+
+  onUploadSuccess(fileArray: Array<any> = []): void {
+    this.files.push(fileArray);
+  }
+
+  onFileRemoved(removedFile: object) {
+    this.files = this.files.filter(file => file[0] !== removedFile);
+  }
+
+  onSubmit() {
+    if (this.validate()) {
```

Comment:
maybe it make sense to create this form using FormControls?

Reply:
we've spoken about this one, it's easier to use ngModel for confirm messages